pipeline.sh

```diff
@@ -9,8 +9,16 @@
 #SBATCH --export=NONE
 #SBATCH --get-user-env=L60
 
-SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-source "${SCRIPT_DIR}"/utils/header.sh
+# Retrieve original directory of submitted script.
```

Comment:
Comment is a bit confusing: if you execute the shell script then you are not submitting the script.